configure.ac

```diff
@@ -303,8 +301,36 @@ AC_MSG_RESULT([${xslstylesheetsdir}])
 AC_MSG_CHECKING([git checkout])
 GIT_CHECKOUT="no"
 if test -n "${GIT}" -a -d "${srcdir}/.git"; then
-        AC_DEFINE([HAVE_CONFIG_VERSION_H], [1], [extra version available in config-version.h])
-        GIT_CHECKOUT="yes"
+	AC_DEFINE([HAVE_CONFIG_VERSION_H], [1], [extra version available in config-version.h])
+	GIT_CHECKOUT="yes"
+
+	REVISION_COMMIT="$(${GIT} log -1 --pretty=format:'%h')"
+
+	REVISION_DESCRIPTION="$(${GIT} describe || echo '<version not available>' )"
+	if test "${REVISION_DESCRIPTION}" = "<version not available>"; then
+		REVISION_DESCRIPTION="$(${GIT} describe --tags || echo '<version not available>')"
+	fi
+
+	if test "${REVISION_DESCRIPTION}" != "<version not available>"; then
+		NUMBER_OF_COMMITS="$(echo ${REVISION_DESCRIPTION} | sed -E 's/^.*-([[1-9]][[0-9]]+)-g[[A-Za-z0-9]]+/\1/')"
+
+		if test "${REVISION_DESCRIPTION}" = "${NUMBER_OF_COMMITS}"; then
+			NUMBER_OF_COMMITS="$(echo ${REVISION_DESCRIPTION} | sed -E 's/^.*-([[1-9]][[0-9]]+)/\1/')"
+		fi
+
+		if test "${REVISION_DESCRIPTION}" = "${NUMBER_OF_COMMITS}"; then
+			NUMBER_OF_COMMITS=0
+		fi
+	else
+		NUMBER_OF_COMMITS=0
+	fi
+
+	COMMIT_DATE="$(${GIT} log -1 --pretty=fuller --date=iso | grep CommitDate: | sed -E 's/^(CommitDate:\s.*)\s+.*/\1/')"
```

Comment:
You've took the commit inside the branch dedicated for release preparation, and naturally it took the latest annotated tag. This commit is not in _master_ branch, so, such situation do not concerns builds on _master_.

Once more,  the rc/prep/rtm tags have not be done directly on master branch, but on the branches dedicated to prepare release. In this terms the actual _v0.16.0-pre1_ is misleading,  but it will by fixed after next release.